lib/urbanopt/geojson/geo_file.rb

```diff
@@ -95,6 +98,9 @@ def self.from_file(path)
               if feature[:properties][:name].nil?
                 raise("No name found for Building Feature")
               end
+              if feature[:properties][:number_of_stories].nil?
+                @@logger.warn("Number of stories is required for feature ID #{feature[:properties][:id]} to calculate if it is shading other building features using the UrbanGeometryCreation measure")
```

Comment:
can you tweak the wording just a bit to this:

@@logger.warn("Number of stories is required to calculate shading using the UrbanGeometryCreation measure...ignoring #{feature[:properties][:id]} in shading calculations")